### ISSUE TITLE
It only support Classification task？？

### ISSUE THREAD

**Issue by Johnson-yue** [automl/Auto-PyTorch#2]
hi，auto-pytorch is only support classification task？？ 
If I have some special issue and some loss function， can I use this for find hyper-parameter like classification task？

**Comment by mlindauer** [comment]
@Johnson-yue Thank you for your interest in AutoPyTorch.

Unfortunately, our main developer is sick this week. So, we will come back to your request next week.

Cheers,
Marius

**Comment by urbanmatthias** [comment]
Hi @Johnson-yue ,

Auto-PyTorch supports classification (binary/multiclass/multilabel) and regression with featurized data.

You can specify the loss function for training the neural networks and a metric (called train_metric, for example accuracy) used by the hyperparameter optimization tool BOHB to evaluate the networks.

Cheers,

Matthias

**Comment by Johnson-yue** [comment]
If I want to train a encoder-decoder model like auto-encoder，can I use it?

**Comment by urbanmatthias** [comment]
No, unforunately this is currently not supported and we won't be able to work on that anytime soon.

If you like to extend Auto-PyTorch, we will be happy to accept pull requests.

Cheers,
Matthias

**Comment by Johnson-yue** [comment]
thank you very much!
